### ISSUE TITLE
question about learning rate and rotate strategy

### ISSUE THREAD

**Issue by CoinCheung** [gidariss/FeatureLearningRotNet#11]
Hi, 

Thanks for providing this awesome work to us!!! 

After reading the code, I am not sure whether I have fully understood it, so I feel I better open an issue to ask: 

1. the original cifar-10 is trained with learning-rate of 0.1 when the batchsize is 128. With the rotnet method, the batchsize is  amplified to 512 (128x4) but the learning rate is still kept 0.1, is that right ?

2. I see in the paper that the strategy of "simultaneously rotate the input image by 4 degrees and enlarge the batchsize 4 times" outperforms "randomly choose one degree to rotate and kept the batchsize not changed". Will the "randomly choose method" bring a significantly bad result, or it is only slightly outperformed by the proposed "4 rotates method" ?

I would be very happy to have your rely. Would you please show me your ideas on these details?

**Comment by gidariss** [comment]
Hi CoinCheung,

1. Yes.
2. No, it is not significantly worse. You might need to increase however the number of training epochs.

**Comment by CoinCheung** [comment]
Thanks for support !!
